styles.css

```diff
@@ -161,6 +160,296 @@
+    font-family: Inter, 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+  }
+
+  /* About Me */
+
+  .aboutme {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 64px 24px;
+    gap: 58px;
+    position: absolute;
+    width: 100%;
+    height: 1522px;
+    left: 0;
+    background-image: url(./assets/AbouyMeBg\ 2.png);
+    background-repeat: no-repeat;
+    background-size: fit;
+    background-position: right top;
+  }
+
+  .aboutme .aboutme-info .aboutme-heading {
+    width: 100%;
+    height: 48px;
+    font-family: Inter, 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    align-items: center;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  .aboutme .aboutme-info .aboutme-text {
+    width: 100%;
+    height: 194px;
+    font-family: Inter, 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    align-items: center;
+    text-align: center;
+    color: #42526e;
+  }
+
+  .aboutme .aboutme-info .resume-btn-container {
+    background-image: url(./assets/AboutMeBg\ 1.png);
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 72px;
+    display: flex;
+    justify-content: center;
+    margin-left: -25px;
+  }
+
+  .aboutme .aboutme-info .resume-btn-container .resume-btn {
+    background-color: #36b37f;
+    width: auto;
+    height: 48px;
+    border-radius: 10px;
+    text-align: center;
+    padding: 12px;
+    color: white;
+    margin-left: 25px;
+    font-family: Inter, 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+  }
+
+  .knowledge {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 1px;
+    gap: 24px;
+    width: 100%;
+    height: 998px;
+    list-style: none;
+  }
+
+  .knowledge .knowledge-tile {
+    width: 100%;
+    height: 316px;
+    background-color: #ebf0ee;
+    border-radius: 10px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    border: 1px solid #c1c7d0;
+    gap: 24px;
+    box-shadow: 0 4px 4px rgba(0, 0, 0, 0.5);
+  }
+
+  .knowledge .knowledge-tile .knowledge-tile-img {
+    width: 80px;
+    height: 80px;
+    background-color: #fff;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-radius: 50%;
+  }
+
+  .knowledge .knowledge-tile .knowledge-title {
+    width: 100%;
+    height: 32px;
+    font-family: Inter, 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    text-align: center;
+    line-height: 32px;
+    color: #344563;
+  }
+
+  .knowledge .knowledge-tile ul {
+    display: flex;
+    flex-direction: row;
+    gap: 8px;
+    list-style: none;
```

Comment:
- [ ] Kindly set the right width for these list elements. As you can see below it is overlapping its container. 
![image](https://user-images.githubusercontent.com/91486791/235228871-601e66f5-b73a-46c9-8977-15e4474f3e14.png)
figma: 
![image](https://user-images.githubusercontent.com/91486791/235228946-57796793-3a5c-462e-aaa0-fa1d4420204b.png)

Reply:
Here is it okay to use fixed widths?